unittests/test_parsers.py

```diff
@@ -59,3 +59,25 @@ def test_file_existence(self):
                             os.path.isfile(importer_test_file),
                             f"Unittest of importer '{importer_test_file}' is missing or using different name"
                         )
+        for parser_dir in os.scandir(os.path.join(basedir, 'dojo', 'tools')):
+            if parser_dir.is_dir():
+                for file in os.scandir(os.path.join(basedir, 'dojo', 'tools', parser_dir.name)):
+                    if file.is_file() and file.name != '__pycache__' and file.name != "__init__.py":
+                        f = os.path.join(basedir, 'dojo', 'tools', parser_dir.name, file.name)
+                        import subprocess
+                        task = subprocess.Popen(["cat", f], stdout=subprocess.PIPE)
+                        read_true = False
+                        for line in task.stdout:
```

Comment:
Simple opening and reading of the file is definitely easier than running a subprocess with `cat`.

Reply:
This is what I tried. But, if you open a file with .read() or readlines() you get an Exception as soon as you hit a line which has .decode("utf-8") in it. This way was the only way I found to get back the content of each line.